src/coreclr/vm/util.cpp

```diff
@@ -177,10 +180,14 @@ void PrintToStdErrA(const char *pszString) {
     }
     CONTRACTL_END
 
-    HANDLE  Handle = GetStdHandle(STD_ERROR_HANDLE);
-
+#if defined(TARGET_ANDROID)
+    // TODO: make priority configurable?
```

Comment:
Maybe it could just be `ANDROID_LOG_ERROR`?

Reply:
Yes.